cypress/integration/demo_tests/demo_spec.js

```diff
@@ -0,0 +1,60 @@
+//Demo OpenSearch-Dashboards Test
+//Note: Getting ResizeObserver loop limit exceeded errors!
+//    Added code from https://github.com/quasarframework/quasar/issues/2233#issuecomment-678115434 
+//    to support/index.js to prevent from failing tests with the error.
+
+describe('opening OpenSearch-Dashboards', () =>{
+    
+    //Setup the dashboard to start at the home page
+    before(() =>{
+        cy.visit('localhost:5601')
```

Comment:
NIT: Probably out of scope and why you used hard-coded strings  but Cypress allows for environment variables and has some settings predefined. Might save you some headaches in the future for any CI if you plan on doing that the earlier you introduce it [[SOURCE](https://docs.cypress.io/guides/guides/environment-variables#Setting)]

Reply:
Implemented environment variables